.github/workflows/test_installer.yml

```diff
@@ -1,75 +1,181 @@
-name: Check Installer
+name: Check Installer (scheduled)
 
 on:
   push:
   pull_request:
   workflow_dispatch:
+  schedule:
+    - cron: '30 5 * * 1'
 
 jobs:
-  install-linux:
+  install-mss-on-linux-from-scratch:
     runs-on: ubuntu-latest
     
     defaults:
       run:
         shell: bash
 
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v3
 
-      - name: Install conda+MSS
+      - name: Install mambaforge+MSS
         run: |
-          sudo rm -rf $(which conda)
+          rm -rf $(which mamba)
```

Comment:
I do not understand the intent of this. Simply removing the binary seems like a bad idea, in general. Can you8 add a comment about the intent of this, as it is not obvious, at least to me?
In particular as this is a fresh Ubuntu, mamba should not be present?

Reply:
lets check this, I think I kept it because it was there.